arviz/plots/jointplot.py

```diff
@@ -20,6 +20,7 @@ def plot_joint(
     fill_last=True,
     joint_kwargs=None,
     marginal_kwargs=None,
+    ax_tuple=None,
```

Comment:
Maybe `axes`?

Reply:
I'd rather `ax` like the other functions or something clearly different like I tried. It it is okey I'll change it by `ax`

Reply:
It is conceptually different than the other functions but you can always overlay plots doing:

    ax = plot_xyz(data1)
    az.plot_xyz(data2, ax=ax)

so maybe for the end users there is not much difference